pkg/controller/certificaterequests/sync_test.go

```diff
@@ -246,7 +249,11 @@ func TestSync(t *testing.T) {
 				},
 			},
 			Builder: &testpkg.Builder{
-				CertManagerObjects: []runtime.Object{gen.CertificateRequest("test")},
+				CertManagerObjects: []runtime.Object{gen.Issuer("ca-issuer",
+					gen.SetIssuerCA(cmapi.CAIssuer{SecretName: "root-ca-secret"}),
```

Comment:
In the previous version of this test, the Issuer also had a `Ready: true` condition set. Was this removed intentionally?